tests/unit/App.spec.ts

```diff
@@ -876,12 +843,13 @@ describe('Dissolution BEN - External User', () => {
     localVue.filter('currency', (x) => x)
     const router = mockRouter.mock()
     router.push({ name: 'dissolution-define-dissolution', query: { id: 'BC0870803' } })
+
     wrapper = mount(App, {
       localVue,
       store,
       router,
       vuetify,
-      stubs: { Affix: true, SbcHeader: true }
+      stubs: { Affix: true, SbcHeader: true, SbcFeeSummary: true }
```

Comment:
I added SbcFeeSummary as a stub so that it doesn't get rendered and make network calls to fetch fees, etc ... we don't care about that in THIS test suite. I created a new test suite for SbcFeeSummary.

Reply:
Note that by stubbing this, I could remove the mock GET calls above.